resources/schemas/dbscripts/sqlserver/targetedms-19.17-19.18.sql

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+DROP TABLE IF EXISTS targetedms.QCEmailNotifications;
```

Comment:
Need to use our stored proc to do this, as this syntax isn't supported in SQL Server 2012, which we still support.

Reply:
changed in the same script.